frontend/src/App.js

```diff
@@ -6,7 +6,7 @@ import SignUp from './components/SignUp/SignUp.js';
 import UsersAll from './components/AdminControls/UsersAll.js';
 import User from './components/AdminControls/User.js';
 import history from './history.js';
-import RecipeList from './components/recipeCard/recipeCard.js';
+import RecipeList from './components/RecipeCard/RecipeCard.js';
```

Comment:
@mglavey in order for this to work, you would need to change the component folder name to 'RecipeCard' & the js file to 'RecipeCard.js' 

this type of minor syntax stuff drives me pretty crazy sometimes

Reply:
Yep, that's what I've been working on. Unfortunately it won't let me commit those changes so I was working to change the folder name manually on github.

Reply:
https://stackoverflow.com/questions/31861651/how-to-rename-a-directory-folder-in-githubweb

Reply:
try 
```git mv recipeCard tmp```
```git mv tmp RecipeCard```

from https://stackoverflow.com/questions/11183788/in-a-git-repository-how-to-properly-rename-a-directory